antarest/study/service.py

```diff
@@ -206,6 +224,54 @@ def assert_raw(study: Study) -> RawStudy:
     return study
 
 
+class StudySettingsDTO(AntaresBaseModel):
+    time_series: TimeSeriesConfiguration
+    general: GeneralConfig
+    advanced_parameters: AdvancedParameters
+    adequacy_patch: AdequacyPatchParameters
+    optimization: OptimizationPreferences
+    thematic_trimming: ThematicTrimming
+    playlist: Playlist
+
+
+class StudyXpansionDTO(AntaresBaseModel):
+    settings: XpansionSettings
+    candidates: list[XpansionCandidate]
+    constraint: bytes
```

Comment:
Can't we have a real model instead of bytes here ?
I don't know what's the actual content of that, but we should as much as possible have business-meaningful objects here, not raw data that are dependent on implementation details (exact format of the data etc).

I think it can be multiple constraint**s**, no ?